Formula/gdal2.rb

```diff
@@ -1,24 +1,21 @@
 class Gdal2 < Formula
   desc "GDAL: Geospatial Data Abstraction Library"
-  homepage "http://www.gdal.org/"
-  url "http://download.osgeo.org/gdal/2.3.2/gdal-2.3.2.tar.gz"
-  sha256 "7808dd7ea7ee19700a133c82060ea614d4a72edbf299dfcb3713f5f79a909d64"
+  homepage "https://www.gdal.org/"
+  url "https://github.com/OSGeo/gdal/archive/v2.3.2.tar.gz"
+  sha256 "e0f751bff9ba6fb541065acbe7a76007be76a3c6309240faf4e6440f6ff1702a"
```

Comment:
My main reason was it seems to have a delay for change to appear in SVN. because the dev is happening on GitHub I'm not sure if the svn support is going to last long @nickrobison

Reply:
I also consider that it is better to use GitHub

Reply:
@fjperini @vincentsarago I agree the we should move from SVN to the github repo for HEAD installations. I was more commenting on moving from download.osgeo.org to GitHub.com. But I'll go ahead and merge this and update the other formulae at a later date.